test/torchaudio_unittest/common_utils/case_utils.py

```diff
@@ -7,7 +7,6 @@
 
 import torch
 from torch.testing._internal.common_utils import TestCase as PytorchTestCase
-import torchaudio
```

Comment:
The test is failing simply because this import statement is being removed. This should stay.

Reply:
That was my bad. Thanks

Reply:
![image](https://user-images.githubusercontent.com/4648995/111080236-d29c8f00-84fd-11eb-8dad-6e6f22e2b9fd.png)

Somehow the style check fails for this.

Reply:
oh, I see. Since now we remove the `'sox' not in torchaudio.list_audio_backends()`, `torchaudio` is no longer used. Yes, this can be finally removed.